tc-server/src/main/java/com/tc/stats/api/DSOMBean.java

```diff
@@ -38,6 +39,8 @@
 
   ObjectName[] getClients();
 
+  String getClientsInformation() throws IOException;
```

Comment:
Instead of adding this method which seems to be single use, it might be better to add the version info to the Client MBean and iterate with the results from getClients()

Reply:
Added getVersion/Revision to the Client MBean, deriving values from minfo.  
Added getConnectedClients() to DSO MBean to access all clients' details.
Moved getClientsInformation to TCServerInfo.